lib/systems/inspect.nix

```diff
@@ -17,6 +17,10 @@ rec {
     isAarch32      = { cpu = { family = "arm"; bits = 32; }; };
     isAarch64      = { cpu = { family = "arm"; bits = 64; }; };
     isMips         = { cpu = { family = "mips"; }; };
+    isMips32       = { cpu = { family = "mips"; bits = 32; }; };
+    isMips64       = { cpu = { family = "mips"; bits = 64; }; };
+    isMips64n32    = { cpu = { family = "mips"; }; abi = { abi = "n32"; }; };
```

Comment:
Should this have an `bits = 64;` for the CPU?

Reply:
For clarity, yes it should; thanks.  Implemented.